src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationProvider.cs

```diff
@@ -77,11 +77,21 @@ private async Task ReloadAsync(CancellationToken cancellationToken)
             }
         }
 
-        private static bool IsJson(string str)
+        private static bool TryParseJson(string data, out JToken? jToken)
         {
+            jToken = null;
+
+            var trimmedData = data.TrimStart();
+            var firstChar = trimmedData.FirstOrDefault();
+
+            if (firstChar != '[' && firstChar != '{')
+            {
+                return false;
+            }
+
             try
             {
-                _ = JToken.Parse(str);
+                jToken = JToken.Parse(trimmedData);
```

Comment:
```suggestion
                jToken = JToken.Parse(data);
```

Reply:
Hey just want to make a comment out of experience so far since you're doing work on whitespaces - in case of copy pasting values to secrets, some times pasting adds a tab (4 spaces) which then results in a "\t" inside the json string and breaks reading of secrets, thought of sharing this if you consider it something worth handling inside the package.

Reply:
@stKris `TrimStart` will cover leading `\t` too. https://dotnetfiddle.net/hNAVqj

Reply:
> Hey just want to make a comment out of experience so far since you're doing work on whitespaces - in case of copy pasting values to secrets, some times pasting adds a tab (4 spaces) which then results in a "\t" inside the json string and breaks reading of secrets, thought of sharing this if you consider it something worth handling inside the package.

Thanks, our fix will handle your case, because the "\t" symbol will be also removed after the trimming.